DEPENDENCIES.md

```diff
@@ -32,11 +32,11 @@
 
 | **Component**      | **Version**       | **Purpose**                                                                 | **Comments**                                                                                                                                                     |
 |--------------------|-------------------|-----------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| oc (OpenShift CLI) | 4.10              | To interact with the cluster                                                | Follows OpenShift version                                                                                                                                        |
+| oc (OpenShift CLI) | 4.11              | To interact with the cluster                                                | Follows OpenShift version                                                                                                                                        |
 | kubectl            | 1.24.x            | To interact with the cluster                                                | Follows kubernetes version which follows OpenShift version. We only need either oc or kubectl                                                                    |
 | kcp plugin         | 0.8.0             | To interact with the kcp workspace                                          | Follows kcp api/server version                                                                                                                                   |
 | tkn pac plugin     | 0.10.0            | To set up PaC                                                               | Optional plugin for customers during the cluster setup phase. Follows PaC Version                                                                                |
-| Argo CD (client)   | 2.3.5             | To run Argo CD related commands                                              | Not recommended for users to use Argo CD to talk to the Argo server instances. Argo is simply used as a model for management of resources using GitOps principles |
+| Argo CD (client)   | 2.3.7             | To run Argo CD related commands                                              | Not recommended for users to use Argo CD to talk to the Argo server instances. Argo is simply used as a model for management of resources using GitOps principles |
```

Comment:
- Hmm, the comment on this is not quite right. ckcp script does require the correct argocd client to be available. Please update the comment.
- Alos, change 2.3.7 to >=2.3.7

Reply:
In the end I have updated argocd CLI version to follow argocd engine version used in openshift gitops(2.4.5).
I guess this is better then using an arbitrary one.